pyvista/core/dataset.py

```diff
@@ -35,17 +35,14 @@
 from .utilities.helpers import is_pyvista_dataset
 from .utilities.misc import abstract_class
 from .utilities.points import vtk_points
-from .utilities.transform import Transform
 
 if TYPE_CHECKING:  # pragma: no cover
     from collections.abc import Callable
     from collections.abc import Generator
     from collections.abc import Iterator
 
     from ._typing_core import MatrixLike
-    from ._typing_core import Number
```

Comment:
`scale` was the only method to use this type. This was changed to `float` as part of the move and so `Number` is no longer imported. I can revert this and keep `Number` if desired.

Reply:
Reverted in [72bc225](https://github.com/pyvista/pyvista/pull/6799/commits/72bc2256fd937e1f80c86693e50417ab577d3530)